vmdb/app/models/metric/capture.rb

```diff
@@ -37,7 +37,7 @@ def self.capture_threshold(target)
   def self.perf_capture_health_check(zone = nil)
     log_header = "MIQ(#{self.name}.perf_capture_health_check)"
     zone ||= MiqServer.my_server.zone(true)
-    q_items = MiqQueue.all(:select => "created_on, args", :conditions => {:state => "ready", :role => "ems_metrics_collector", :method_name => "perf_capture", :zone => zone.name}, :order => "created_on ASC")
+    q_items = MiqQueue.select("created_on, args").where(:state => "ready", :role => "ems_metrics_collector", :method_name => "perf_capture", :zone => zone.name).order("created_on ASC")
```

Comment:
Weird...I saw it as two separate params...not sure why I commented on this one. :eyes_glazing_over: